site/profile/manifests/core/icinga_agent.pp

```diff
@@ -2,95 +2,99 @@
 #   Icinga agent creation for metric collections
 
 class profile::core::icinga_agent(
+  Boolean $install,
```

Comment:
I don't like this approach. It would be better to add a boolean param to `profile::core:common` as I suggested in review to turn inclusion of this class on and off.  The advantage of that is only needing one key to turn the agent behavior on and off instead of having to define a bunch of unused keys per site.

Reply:
done!